content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/spawning.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Spawning Autonomous Player Entities
+description: A reference for spawning and registering network player entities
+linktitle: Spawning Players
+---
+
+When a Player join a Multiplayer session, one of the first things that often occurs is spawning a networked entity for them to control. In order to do this, the Multiplayer Gem provides an interface to specify an entity for a Player and have the Gem mark it as Autonomous (as defined in (./overview)).
+
+*IMultiplayerSpawner* is an AZ::Interface<T> that provides a mechanism to tell the Multiplayer Gem what to spawn and where when a Player Joins. It also provides a hook to clean up when a Player Leaves. It is intended to implemented by games using the Multiplayer Gem. 
```

Comment:
docs team might say "leave" and "join" be lowercase here.

Reply:
```suggestion
*IMultiplayerSpawner* is an `AZ::Interface<T>` that provides a mechanism to tell the multiplayer gem what to spawn and where when a player joins a session. `IMultiplayerSpawner` also provides a hook to clean up when a player leaves. All multiplayer games should provide an implementation to handle the events delivered for player join/player leave events.
```

This interface is not present in 2111.2 - I'm assuming that these docs are for incoming development work. Right now there is a docs proposal open for splitting `main` and `development` into their own separate docs workflows ([RFC](https://github.com/o3de/sig-docs-community/issues/32), closes 3/8). They may be put on hold for merge as a result.

Otherwise, we'd recommend linking directly to the API reference for any mentioned class or method.